src/Package/Impl/Sql/Publish/SqlPublshOptionsDialog.xaml.cs

```diff
@@ -35,7 +35,7 @@ public partial class SqlPublshOptionsDialog : PlatformDialogWindow {
 
         public static async Task<SqlPublshOptionsDialog> CreateAsync(
             ICoreShell shell, IProjectSystemServices pss, IProjectConfigurationSettingsProvider pcsp, ISettingsStorage settings) {
-            var dialog = await CreateAsync(shell, pss, new FileSystem(), pcsp, settings);
+            var dialog = await CreateAsync(shell, pss, new WindowsFileSystem(), pcsp, settings);
```

Comment:
`fs` parameter isn't used in `SqlPublshOptionsDialog` - you can reduce the whole thing to one constructor